Pull-requests-at-Oppia.md

```diff
@@ -0,0 +1,64 @@
+## Table of contents
+
+* [PR lifecycle](#pr-lifecycle)
+* [Draft pull requests](#draft-pull-requests)
+* [Labeling issues and pull requests](#labeling-issues-and-pull-requests)
+* [Juggling multiple pull requests](#juggling-multiple-pull-requests)
+
+## Pull request lifecycle
+
+![Flowchart of PR lifecycle](images/prLifecycle.png)
+
+<details>
+<summary>Flowchart source code</summary>
+
+The flowchart was generated from the below source code by https://flowchart.fun.
+
+```text
+~~~
+layout:
+  name: dagre
+  rankDir: LR
+~~~
+Draft PR
+  Ready for review: PR awaiting review
+    Request changes: Changes requested
+      Addresses comments: (PR awaiting review)
+    All reviewers approve: PR labeled LGTM
+      Merge: PR closed
+```
+</details>
+
+1. (optional) Draft PR: Authors can open draft PRs to get early feedback or debug CI tests. Reviews at this stage are at the discretion of the author.
```

Comment:
Perhaps elaborate more on what "Reviews at this stage are at the discretion of the author" means. In particular what is the author supposed to do / what decision are they supposed to make here, and how do they communicate this decision?

Reply:
done